read-the-docs-500MM-pageviews.rst

```diff
@@ -0,0 +1,106 @@
+.. post:: July 27, 2016
+   :tags: report
+   :author: Scott
+
+Read the Docs Hits 500MM Pageviews
```

Comment:
Should we spell out Million here? Or use the full `500,000,000` because it looks more impressive? :)